app/src/androidTest/java/com/github/mateo762/myapplication/profile/ProfileActivityTestWithIntent.kt

```diff
@@ -0,0 +1,71 @@
+package com.github.mateo762.myapplication.profile
```

Comment:
Why to distinguish between tests with intent and the other ones?

Reply:
We have different flows, if-else branches depending on whether there is an intent, so we need to test each flow

Reply:
Is there a way to make it more maintainable?